yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -470,46 +492,68 @@ defaultRegisterHandler = do
                 fsName = Just "email",
                 fsAttrs = [("autofocus", "")]
             }
+                passwordSettings = FieldSettings {
+                fsLabel = SomeMessage Msg.Password,
+                fsTooltip = Nothing,
+                fsId = Just "password",
+                fsName = Just "password",
+                fsAttrs = [("placeholder", "optional")]
+            }
 
             (emailRes, emailView) <- mreq emailField emailSettings Nothing
+            (passwordRes, passwordView) <- mopt passwordField passwordSettings Nothing
 
             let userRes = UserForm <$> emailRes
             let widget = do
                 [whamlet|
                     #{extra}
                     ^{fvLabel emailView}
-                    ^{fvInput emailView}
+                    <div>
+                        ^{fvInput emailView}
+                    <div>
+                        ^{fvInput passwordView}
```

Comment:
I'd feel a bit better if the display of a password field were completely optional, allowing people with current behavior to remain with the current behavior entirely.